mopidy/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM debian:stretch-slim
+
+RUN set -ex \
+    # Official Mopidy install for Debian/Ubuntu along with some extensions
+    # (see https://docs.mopidy.com/en/latest/installation/debian/ )
+ && apt-get update \
+ && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        curl \
+        dumb-init \
+        gcc \
+        gnupg \
+        gstreamer1.0-alsa \
+        gstreamer1.0-plugins-bad \
+        python-crypto \
+        python-pykka \
+ && curl -L https://apt.mopidy.com/mopidy.gpg | apt-key add - \
+ && curl -L https://apt.mopidy.com/mopidy.list -o /etc/apt/sources.list.d/mopidy.list \
+ && apt-get update \
```

Comment:
Be interesting to see what this adds. The Pi version for instance adds:
```
pi@mopidy-staging:~ $ cat /etc/apt/sources.list.d/mopidy.list
# Mopidy APT archive
# Built on Debian 10 (buster)
deb http://apt.mopidy.com/ buster main contrib non-free
deb-src http://apt.mopidy.com/ buster main contrib non-free
```
hence the `buster` comment